Refit/IRefitResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+namespace Refit
+{
+    public interface IRefitResponse<T>
```

Comment:
I don't think there's a need for this interface. It doesn't seem to serve any real purpose -- you'd never want to mock it or inject another implementation. so it's just adding noise IMO.

Reply:
Yeah, probably no need for that.

Reply:
@bennor  @onovotny Removing the interface as suggested.